specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-04-01-preview/machineLearningServices.json

```diff
@@ -1228,6 +1228,284 @@
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "LinkedWorkspaces"
+        ],
+        "operationId": "LinkedWorkspaces_Create",
+        "description": "Add a new linked workspace.",
+        "x-ms-examples": {
+          "CreateLinkedWorkspace": {
+            "$ref": "./examples/createLinkedWorkspace.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The object for adding a new linked workspace",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/LinkedWorkspaceCreateDto"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully created the linked workspace.",
+            "schema": {
+              "$ref": "#/definitions/LinkedWorkspace"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/Microsoft.MachineLearningServices/workspaces/{workspaceName}/linkedWorkspaces/{linkName}": {
+      "get": {
+        "tags": [
+          "LinkedWorkspaces"
+        ],
+        "operationId": "LinkedWorkspaces_Get",
+        "description": "Get the detail of a linked workspace.",
+        "x-ms-examples": {
+          "GetLinkedWorkspace": {
+            "$ref": "./examples/getLinkedWorkspace.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/LinkName"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the detail of the linked workspace.",
+            "schema": {
+              "$ref": "#/definitions/LinkedWorkspace"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      },
+      "patch": {
```

Comment:
Just FYI, PATCH is not strictly required by RPC on proxyOnly entities, but it is nice to have. Will the service support it?

Reply:
in this case, we'll remove it.
Not sure what do you mean by "proxyOnly" entity, is the one in manifest file with "proxyOnly" as routingType? Please note that as we mentioned in the review meeting, LinkedWorkspace is not registered in the manifest, what do you call this entity?

Reply:
Yes - that's ProxyOnly. Per ARM contract, the only _required_ PATCH apis are to support patch for tags. Only "tracked" resources have tags. So, it's not strictly required to have PATCH method on ProxyOnly resource. If the service will support it, it does not need to be removed.

Reply:
Patch is no difference with Put, so will remove it.